services/time/src/lib.rs

```diff
@@ -35,381 +35,41 @@ extern crate serde_derive;
 #[macro_use]
 extern crate serde_json;
 
-use iron::{Handler, prelude::*};
-use router::Router;
-use chrono::{DateTime, Duration, TimeZone, Utc};
-
-use std::sync::{Arc, RwLock};
-
-use exonum::blockchain::{ApiContext, Blockchain, ExecutionError, ExecutionResult, Schema, Service,
-                         ServiceContext, Transaction, TransactionSet};
-use exonum::messages::{Message, RawTransaction};
-use exonum::encoding::serialize::json::reexport::Value;
-use exonum::storage::{Entry, Fork, ProofMapIndex, Snapshot};
-use exonum::crypto::{Hash, PublicKey};
+use exonum::api::Api;
+use exonum::blockchain::{ApiContext, ExecutionError, Service, ServiceContext, Transaction,
+                         TransactionSet};
+use exonum::crypto::Hash;
 use exonum::encoding;
+use exonum::encoding::serialize::json::reexport::Value;
 use exonum::helpers::fabric::{Context, ServiceFactory};
-use exonum::api::Api;
+use exonum::messages::RawTransaction;
+use exonum::storage::{Fork, Snapshot};
 
+use iron::Handler;
+use router::Router;
+use schema::TimeSchema;
+use time_provider::{SystemTimeProvider, TimeProvider};
+use transactions::*;
+
+/// Node API.
+pub mod api;
+/// Database schema.
+pub mod schema;
+/// Node transactions.
+pub mod transactions;
+/// System time provider.
+pub mod time_provider;
 /// Time service id.
-const SERVICE_ID: u16 = 4;
+pub const SERVICE_ID: u16 = 4;
```

Comment:
Well, I think we need to keep it public, so user would be able to see actual value in docs.